tests/integration/backward_compatible/proxy/reliable_topic_test.py

```diff
@@ -85,6 +88,8 @@ def test_add_listener(self):
 
         messages = []
 
+        on_cancel_call_count = [0]
```

Comment:
Not sure about synchronization of this list, so I think using a AtomicInteger is better. See other tests, e.g ConnectionManagerOnClusterRestartTest

Reply:
fixed at 1e850636a0d7ab69484387cdf6516ef843d85b3d